runtime/doc/support.txt

```diff
@@ -13,12 +13,13 @@ Supported platforms					 *supported-platforms*
 
 `System`          `Tier`      `Versions`                  `Tested versions`
 Linux            1      >= 2.6.32, glibc >= 2.12     Ubuntu 22.04
-macOS (Intel)    1      >= 10.15                     macOS 12
-Windows 64-bit   1      >= 8 (see note below)        Windows Server 2022
+macOS (Intel)    1      >= 11                        macOS 12
+macOS (M1)       2      >= 11                        macOS 14
```

Comment:
Why do macOS (Intel) and macOS (M1) have different support tiers?

Reply:
Because we only test(ed) on Intel before M1 runners were GA.

Reply:
Yeah, probably just outdated. Send PR

Reply:
(and chocolate.)